### PR TITLE
Simpler graphql names

### PR DESCRIPTION
**Pull request by jeffsee55**
Part 1 of simplifying GraphQL names

`getPostDocument` -> `post`
`getDocument` -> `document`
`getCollections` -> `collections`
`getCollection` -> `collection`

Closes https://github.com/tinacms/product-roadmap/issues/89
Closes https://github.com/tinacms/product-roadmap/issues/90

> Note: this is being merged into a feature branch `simplify-graphql-names`, which will include more changes before merging back to `main`.

### CODE REVIEW

**Comment by logan-anderson** [comment]
Looking good to me. Just left a couple of small comments.

File: experimental-examples/tina-cloud-starter/pages/posts/[filename].tsx [hunk 1]
```diff
@@ -5,8 +5,8 @@ import { ExperimentalGetTinaClient } from "../../.tina/__generated__/types";
 export default function BlogPostPage(
   props: AsyncReturnType<typeof getStaticProps>["props"]
 ) {
-  if (props.data && props.data.getPostsDocument) {
-    return <Post {...props.data.getPostsDocument} />;
```
[the remark this answers is not in the document]
Comment (jeffsee55) [reply experimental-examples/tina-cloud-starter/pages/posts/[filename].tsx:9]: I started doing this and then I realized that having a smaller footprint on these will make rebasing easier. Let's plan to circle back on this one so I don't end up with a ton of conflicts while rebasing

**Comment by jeffsee55** [comment]
> I get the feeling the Sidebar is probably broken with this change as it also makes a call to getCollections:

This is updated internally in the Admin API, I'm trying to leave a small footprint on these changes, so rebasing is easier, but yes, that name could be improved. Will put that in the queue.

**Review by Enigmatical — APPROVED** [review]
All of this looks good to me 👍

**Pull request merged**